### PR TITLE
Fix error raising due to removed to_d method

### PR DESCRIPTION
**Pull request by ysv**

### CODE REVIEW

**Comment by ysv** [comment]
@yivo review please

File: app/models/matching/limit_order.rb [hunk 1]
```diff
@@ -9,11 +9,11 @@ def initialize(attrs)
       @id         = attrs[:id]
       @timestamp  = attrs[:timestamp]
       @type       = attrs[:type].to_sym
-      @volume     = attrs[:volume].to_d
-      @price      = attrs[:price].to_d
+      @volume     = attrs[:volume].blank? ? nil : attrs[:volume].to_d
```
Comment (yivo) [inline app/models/matching/limit_order.rb]: @yssavchuk I am not sure we should return nil. Could you please check how activesupport 4.0 & ruby 2.2 handles `nil.to_d`?

**Review by yivo — CHANGES_REQUESTED** [review]

File: app/models/matching/limit_order.rb (hunk 1 above)
Comment (ysv) [reply app/models/matching/limit_order.rb]: @yivo it throws execption

Comment (ysv) [reply app/models/matching/limit_order.rb]: but the reason of error in this test is "".to_d
```
2.2.0 :003 > "".to_d
 => #<BigDecimal:55765024f678,'0.0',9(9)>
```
```
2.4.2 :003 > "".to_d
ArgumentError: invalid value for BigDecimal(): ""
	from /home/yarpolk/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/bigdecimal/util.rb:61:in `BigDecimal'
	from /home/yarpolk/.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/bigdecimal/util.rb:61:in `to_d'
	from (irb):3
	from /home/yarpolk/.rvm/rubies/ruby-2.4.2/bin/irb:11:in `<main>'
```

Comment (ysv) [reply app/models/matching/limit_order.rb]: @yivo do I need to monkey-patch to_d method which returns 0 ?

Comment (yivo) [reply app/models/matching/limit_order.rb]: @yssavchuk did you check with activesupport 4.0?

**Pull request merged**